mca/mca.py

```diff
@@ -72,19 +72,26 @@ def write_record(self, record_type, data):
         if self.record_data:
             self.record_data.write_record(record_type, data)
 
-    def send_flow_ids_ttl(self, flow_ids, ttl, save=True, check_before=True):
+    def send_flow_ids_ttl(self,
+                          flow_ids: Sequence[Sequence[int]],
```

Comment:
`Sequence[tuple[int]]`

Reply:
I think a `NamedTuple` would be a nice way to handle these tuples.

Reply:
For the `NamedTuple` issue, I'll change it in a separate changeset since that could potentially affect other aspects of the code.